README.md

```diff
@@ -29,35 +29,121 @@ The Class Request Tool (CRT) is a class reservation system that lets instructors
 
 ## Application Set-up Steps
 
-1. Get code from: https://github.com/harvard-library/class_request_tool
-2. Run bundle install. You will probably have to install OS-vendor supplied libraries to satisfy some gem install requirements.
-3. Create the database and create a config/database.yml file modeled on  "[config/database.yml.postgres](config/database.yml.postgres)" to suit your environment.
-4. Run `rake db:schema:load`.
-5. Create a .env file for your environment. Currently, the following variables are needed to run Class Request Tool:
+### Preparation
+1. Clone the code from the github repository
+`git clone https://github.com/harvard-library/class_request_tool`
+2. Create a .env file for your environment. Copy the example text file `env-example.txt` and rename it to `.env`. Read [Environment variables](#environment-variables) and [Database configuration](#database-configuration) for more information.
+3. To add your initial customizations, create a `config/customization.yml` file based on [config/customization.yml.example](config/customization.yml.example), and a `config/mailer.yml.erb` file based on [config/mailer.yml.example](config/mailer.yml.example).
```

Comment:
I'd prefer the customization.yml and config/mailer be separate steps - I think it's easy to trip past the second one when they're in one bullet point.